install.sh

```diff
@@ -56,20 +56,26 @@ then
               #rpm -Uvh http://www.elrepo.org/elrepo-release-7.0-2.el7.elrepo.noarch.rpm
               #yum --enablerepo=elrepo-kernel -y install kernel-ml
               
+              
               # AUTO-CONFIGURE MOUNT OPTIONS
               echo "Auto-Configuring mount options..."
-
-              # Stores name of partitions(space seperated) on which caches are present
-              cache_prt=$(mount | grep -E '/\<cache\> | /cache[[:digit:]]+' | awk '{print $1}')
-              echo "Cache partitions are :"
-              echo $(tput bold)$cache_prt $(tput sgr0)
+              # Stores name of partitions(space seperated) on which caches,logs,data are present
+              cache_prt=$(cat /etc/fstab | grep -E '[[:blank:]]/cache[[:blank:]]|/cache[[:digit:]]+' | awk '{print $2}')
```

Comment:
The regex is a bit ugly but it sure works.
This version only accesses /etc/fstab, it doesn't need mount command to be run.